spec/lib/myfinance/resources/receivable_account_spec.rb

```diff
@@ -161,4 +230,154 @@
+    end
+  end
+
+  describe "#create_as_parcelled", vcr: true do
+    let(:params) { { due_date: '2015-10-20', amount: 150.99, create_as_recurrent: "monthly", number_of_parcels: 7 } }
+    subject { client.receivable_accounts.create_as_parcelled(entity_id, params) }
+
+    context "with success" do
+      it "creates a new object" do
+        expect(subject.id).to eq(1298985)
+        expect(subject.due_date).to eq(Date.new(2015, 10, 20))
+        expect(subject.entity_id).to eq(entity_id)
+        expect(subject.status).to eq(1)
+        expect(subject.status_name).to eq('unreceived')
+        expect(subject.occurred_at).to be_nil
+        expect(subject.amount).to eq(150.99)
+        expect(subject.ticket_amount).to be_nil
+        expect(subject.interest_amount).to be_nil
+        expect(subject.discount_amount).to be_nil
+        expect(subject.total_amount).to be_nil
+        expect(subject.description).to be_nil
+        expect(subject.document).to be_nil
+        expect(subject.document_emission_date).to be_nil
+        expect(subject.observation).to be_nil
+        expect(subject.remind).to be_falsy
+        expect(subject.reminded_at).to be_nil
+        expect(subject.income_tax_relevant).to be_falsy
+        expect(subject.category_id).to be_nil
+        expect(subject.classification_center_id).to be_nil
+        expect(subject.expected_deposit_account_id).to be_nil
+        expect(subject.recurrence_id).to eq(8833)
+        expect(subject.person_id).to be_nil
+        expect(subject.created_at).to eq(DateTime.parse("2015-10-20T16:27:15-02:00"))
+        expect(subject.updated_at).to eq(DateTime.parse("2015-10-20T16:27:15-02:00"))
+        expect(subject.recurrent).to be_falsy
+        expect(subject.parcelled).to be_truthy
+        expect(subject.recurrence_period).to eq("monthly")
+        expect(subject.number_of_parcels).to eq(7)
+        expect(subject.current_parcel).to eq(1)
+        expect(subject.competency_month).to eq("2015-10")
+        expect(subject.financial_account_taxes_attributes).to be_empty
+        expect(subject.links).to eq([
+          {"rel" => "self","href" => "https://sandbox.myfinance.com.br/entities/3798/receivable_accounts/1298985","method" => "get"},
+          {"rel" => "destroy","href" => "https://sandbox.myfinance.com.br/entities/3798/receivable_accounts/1298985","method" => "delete"},
+          {"rel" => "receive","href" => "https://sandbox.myfinance.com.br/entities/3798/receivable_accounts/1298985/receive","method" => "put"},
+          {"rel" => "next","href" => "https://sandbox.myfinance.com.br/entities/3798/receivable_accounts/1298986","method" => "get"},
+          {"rel" => "destroy_recurrence","href" => "https://sandbox.myfinance.com.br/entities/3798/receivable_accounts/1298985/recurrence","method" => "delete"}
+        ])
+      end
+    end
+
+    context "when any data is invalid" do
+      let(:params) { { due_date: '2015-10-20', amount: 150.99, create_as_recurrent: "october" } }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.